app/models/cdr/cdr.rb

```diff
@@ -167,6 +167,8 @@ class Cdr::Cdr < Cdr::Base
 
   scope :account_id_eq, lambda { |account_id| where('vendor_acc_id =? OR customer_acc_id =?', account_id, account_id) }
 
+  scope :where_customer, -> (id) { where(customer_id: id) }
+  scope :where_account, -> (id) { where(customer_acc_id: id) } # OR vendor_acc_id ???
```

Comment:
@dmitry-sinina take a look at this query. This is ony for Customers API, so I did not add `OR vendor_acc_id = ?`, I'm not sure about this.

Reply:
` { where(customer_acc_id: id) }` is right.